prowler/lib/outputs/outputs.py

```diff
@@ -210,6 +210,8 @@ def send_to_s3_bucket(
             filename = f"{output_filename}{json_asff_file_suffix}"
         elif output_mode == "html":
             filename = f"{output_filename}{html_file_suffix}"
+        else:  # Compliance output mode
+            return
```

Comment:
Why this? Why do we don't want to send compliance to S3?

Reply:
Also if you modify this function's behaviour please update the test to handle this case.

Reply:
Done